bitcoin/src/bip152.rs

```diff
@@ -9,7 +9,7 @@ use core::{convert, fmt, mem};
 use std::error;
```

Comment:
The commit message mentions 16 bit but the entire commit is about not supporting more than 64 bit sizes.

Reply:
Oh thanks, I woke up in the night wondering if I had this patch wrong - maybe my brain remembered that I miss described it :)